### PR TITLE
theme: add doom-bluloco-dark and doom-bluloco-light themes

### PR DESCRIPTION
**Pull request by RazikSF**
Both dark and light themes are ported from VS Code's Bluloco themes.

screenshot-doom-blulobo-dark

screenshot-doom-blulobo-light

-----
- [x] I searched the issue tracker and this hasn't been PRed before.
- [x] My changes are not on [the do-not-PR list](https://doomemacs.org/d/do-not-pr) for this project.
- [x] My commits conform to [the git conventions](https://doomemacs.org/d/git-conventions).
- [x] My changes are visual; I've included before and after screenshots.

### CODE REVIEW

**Comment by hankertrix** [comment]
Would love for this to be merged into the repository!

File: README.md [hunk 1]
```diff
@@ -8,19 +8,19 @@
 
 A theme megapack for GNU Emacs, inspired by community favorites. Special
 attention is given for [Doom Emacs](https://doomemacs.org) and [solaire-mode]
-support, but will work fine anywhere else. 
+support, but will work fine anywhere else.
```
Comment (gagbo) [inline README.md:11]: Is it possible to revert the formatting changes?

File: README.md [hunk 2]
```diff
@@ -8,19 +8,19 @@
...
 
 [See the screenshots.][screenshots]
 
-
 # Table of Contents
+
```
Comment (gagbo) [inline README.md:16]: Is it possible to revert the formatting changes?

File: README.md [hunk 3]
```diff
@@ -8,19 +8,19 @@
...
-    - [Manually](#manually)
+  - [Doom Emacs](#doom-emacs)
+  - [Manually](#manually)
 - [Theme list](#theme-list)
-    - [Flagship themes](#flagship-themes)
-    - [Additional themes](#additional-themes)
-    - [Planned themes](#planned-themes)
+  - [Flagship themes](#flagship-themes)
+  - [Additional themes](#additional-themes)
+  - [Planned themes](#planned-themes)
```
Comment (gagbo) [inline README.md:23]: Is it possible to revert the formatting changes?

File: README.md [hunk 4]
```diff
@@ -33,7 +33,7 @@ support, but will work fine anywhere else.
 The built-in `:ui doom` module installs and configures `doom-themes` for you,
 and loads `doom-one` by default. To change the theme, change `doom-theme`:
 
-``` emacs-lisp
+```emacs-lisp
```
Comment (gagbo) [inline README.md:36]: Is it possible to revert the formatting changes?

File: README.md [hunk 5]
```diff
@@ -63,22 +63,23 @@ common defaults laid out:
   (doom-themes-org-config))
 ```
 
-
 ## Theme list
```
Comment (gagbo) [inline README.md:66]: Is it possible to revert the formatting changes?

File: README.md [hunk 6]
```diff
@@ -63,22 +63,23 @@ common defaults laid out:
...
 
 This megapack's first and flagship theme is `doom-one` (inspired by Atom One
 Dark), but also includes 67 themes submitted to us by the Emacs community. We
 welcome PRs to help us maintain and address inconsistencies in them.
 
 | Name                              | Source                                                                                 | Description                                                                                                                       |
-|-----------------------------------|----------------------------------------------------------------------------------------|-----------------------------------------------------------------------------------------------------------------------------------|
+| --------------------------------- | -------------------------------------------------------------------------------------- | --------------------------------------------------------------------------------------------------------------------------------- |
```
Comment (gagbo) [inline README.md:73]: Is it possible to revert the formatting changes?

File: README.md [hunk 7]
```diff
@@ -125,7 +126,7 @@ welcome PRs to help us maintain and address inconsistencies in them.
...
 | doom-palenight                    | [link](https://github.com/equinusocio/vsc-material-theme)                              | adapted from equinusocio's Material themes (ported by [@Brettm12345](https://github.com/Brettm12345))                             |
 | doom-peacock                      | [link](https://github.com/daylerees/colour-schemes/blob/master/emacs/peacock-theme.el) | inspired by daylerees' Peacock (ported by [@teesloane](https://github.com/teesloane))                                             |
-| doom-pine                         | [link](https://github.com/morhetz/gruvbox)                                             | a green-tinged Gruvbox (by [@RomanHargrave](https://github.com/RomanHargrave))                                           |
+| doom-pine                         | [link](https://github.com/morhetz/gruvbox)                                             | a green-tinged Gruvbox (by [@RomanHargrave](https://github.com/RomanHargrave))                                                    |
```
Comment (gagbo) [inline README.md:129]: Is it possible to revert the formatting changes?

File: README.md [hunk 8]
```diff
@@ -144,8 +145,8 @@ welcome PRs to help us maintain and address inconsistencies in them.
 | doom-xcode                        | Xcode.app                                                                              | based off of Apple's Xcode Dark Theme (ported by [@kadenbarlow](https://github.com/kadenbarlow))                                  |
 | doom-zenburn                      | [link](https://github.com/bbatsov/zenburn-emacs)                                       | port of the popular Zenburn theme (ported by [@jsoa](https://github.com/jsoa))                                                    |
 
-
 ## Extensions
+
```
Comment (gagbo) [inline README.md:149]: Is it possible to revert the formatting changes?

File: README.md [hunk 9]
```diff
@@ -156,24 +157,25 @@ extensions][wiki].
...
+  - Make statistic cookies respect underlying faces.
+  - Fontify item bullets (make them stand out)
+  - Fontify item checkboxes (and when they're marked done), like TODOs that
+    are marked done.
+  - Fontify dividers/separators (5+ dashes)
+  - Fontify #hashtags and @at-tags, for personal convenience; see
+    `doom-org-special-tags` to disable this.
+
 ## Complementary plugins
+
```
Comment (gagbo) [inline README.md:178]: Is it possible to revert the formatting changes?

File: README.md [hunk 10]
```diff
@@ -183,62 +185,62 @@ The following plugins complement our themes:
...
      `(mode-line ((t (:background ,(doom-color 'dark-violet)))))
      `(font-lock-comment-face ((t (:foreground ,(doom-color 'base6))))))
    ```
-  
+
 3. Copy your favorite theme into your `custom-theme-directory` (normally
    `~/.emacs.d/`, or `~/.doom.d/themes` for Doom users), and tweak it there.
-   
-  
+
```
Comment (gagbo) [inline README.md:218]: Is it possible to revert the formatting changes?

File: README.md [hunk 11]
```diff
@@ -183,62 +185,62 @@ The following plugins complement our themes:
...
 [city-lights]: http://citylights.xyz/
 [dark+]: https://github.com/microsoft/vscode/blob/master/extensions/theme-defaults/themes/dark_plus.json
 [das-s]: https://github.com/das-s
 [daylerees]: http://daylerees.github.io/
 [defphil]: https://github.com/defphil
-[Dracula]: https://draculatheme.com/
+[dracula]: https://draculatheme.com/
 [elenapan's dotfiles]: https://github.com/elenapan/dotfiles
 [ema2159]: https://github.com/ema2159
 [emacs.d]: https://github.com/hlissner/.emacs.d
```
Comment (gagbo) [inline README.md:246]: Is it possible to revert the formatting changes?

File: README.md [hunk 12]
```diff
@@ -247,44 +249,44 @@ support.
...
 [outrun]: https://github.com/samrap/outrun-theme-vscode
 [paladhammika]: https://github.com/paladhammika
 [plain]: https://github.com/gko/plain
 [sagittaros]: https://github.com/sagittaros/
 [sailorhg]: https://sailorhg.github.io/fairyfloss/
 [screenshots]: https://github.com/doomemacs/themes/tree/screenshots
-[ShaneKilkelly]: https://github.com/ShaneKilkelly
+[shanekilkelly]: https://github.com/ShaneKilkelly
 [shades-of-purple]: https://github.com/ahmadawais/shades-of-purple-vscode
 [snazzy]: https://github.com/sindresorhus/hyper-snazzy
```
Comment (gagbo) [inline README.md:291]: Is it possible to revert the formatting changes?

File: README.md [hunk 13]
```diff
@@ -294,19 +296,19 @@ support.
...
-[JordanFaust]: https://github.com/JordanFaust
+[jordanfaust]: https://github.com/JordanFaust
 [vscode-1337]: https://github.com/microsoft/vscode-themes/tree/main/1337
-[Zenburn]: https://github.com/bbatsov/zenburn-emacs
+[zenburn]: https://github.com/bbatsov/zenburn-emacs
 [eziam]: https://github.com/thblt/eziam-theme-emacs
 [jbeans]: https://github.com/synic/jbeans-emacs
 [tao]: https://github.com/11111000000/tao-theme-emacs
-[FosterHangdaan]: https://github.com/FosterHangdaan
+[fosterhangdaan]: https://github.com/FosterHangdaan
```
Comment (gagbo) [inline README.md:314]: Is it possible to revert the formatting changes?

**Review by gagbo — CHANGES_REQUESTED** [review]
Seems good, there are a just a bunch of formatting changes in the markdown that slipped in

**Comment by RazikSF** [comment]
> Seems good, there are a just a bunch of formatting changes in the markdown that slipped in

Thanks for the reviews. I reverted the markdown to its initial formatting.

**Review by hlissner — APPROVED** [review]

**Review by hlissner — CHANGES_REQUESTED** [review]
Would you mind squashing both 11d3bdd7b62214fff0e941c10cb0cf4a1cd986c9 and 2f8ba5816811f4d27366b3d2aa744d73737c3428 into c15ec12fb7abc9a6c15b64645f345dc303a2a417? Otherwise, this PR looks great! I look forward to merging it.

**Comment by RazikSF** [comment]
> Would you mind squashing both [11d3bdd](https://github.com/doomemacs/themes/commit/11d3bdd7b62214fff0e941c10cb0cf4a1cd986c9) and [2f8ba58](https://github.com/doomemacs/themes/commit/2f8ba5816811f4d27366b3d2aa744d73737c3428) into [c15ec12](https://github.com/doomemacs/themes/commit/c15ec12fb7abc9a6c15b64645f345dc303a2a417)? Otherwise, this PR looks great! I look forward to merging it.

Thank you! Done! I hope it worked.

**Review by hlissner — APPROVED** [review]

**Comment by hlissner** [comment]
LGTM! Thanks for your help!

**Comment by uloco** [comment]
@RazikSF Just wanted to say thank you for porting bluloco 🙏🏼 There is a neovim version too, so if you need any more inspiration you can have a look https://github.com/uloco/bluloco.nvim

**Comment by RazikSF** [comment]
> @RazikSF Just wanted to say thank you for porting bluloco 🙏🏼 There is a neovim version too, so if you need any more inspiration you can have a look https://github.com/uloco/bluloco.nvim

Thank you for your amazing theme @uloco! I will definitely take a look for more inspiration.

**Pull request merged**